subcommand/server-acl-init/rules.go

```diff
@@ -121,6 +127,42 @@ namespace_prefix "" {
 	return c.renderRules(meshGatewayRulesTpl)
 }
 
+func (c *Command) ingressGatewayRules(name, namespace string) (string, error) {
+	ingressGatewayRulesTpl := `
+{{- if .EnableNamespaces }}
+namespace "{{ .GatewayNamespace }}" {
+{{- end }}
+  service "{{ .GatewayName }}" {
+     policy = "write"
+  }
+{{- if .EnableNamespaces }}
+}
+{{- end }}
```

Comment:
I think we need to add
```
node_prefix "" {
  policy = "read"
}
service_prefix "" {
  policy = "read"
}
```

So the ingress can route to other services. It's not crazy permissions since we already have that in the anon token and every envoy sidecar and intentions are still required to actually allow requests to those services.